src/compilers/project.json

```diff
@@ -8,8 +8,8 @@
       "type": "platform",
       "version": "1.0.0-rc2-23931"
     },
-    "Microsoft.CodeAnalysis.CSharp": "1.3.0-beta1-20160329-01",
-    "Microsoft.Net.Compilers.netcore": "1.3.0-beta1-20160329-01",
+    "Microsoft.CodeAnalysis.CSharp": "1.3.0-beta1-20160405-05",
```

Comment:
Last I checked this project gets published into the sdk publish directory.

I'm not sure what it's original intention is but probably a hack to get unblocked in the hackathon.

Checkout out `CompileCliSdk` in CompileTargets.cs